src/api/fullnode.ts

```diff
@@ -39,6 +40,21 @@ export const downloadTx = async (txId: string, noCache: boolean = false) => {
   return response.data;
 };
 
+/**
+ * Returns a list of transactions on the mempool
+ *
+ */
+export const downloadMempool = async () => {
+  const response = await axios.get(`${DEFAULT_SERVER}mempool`);
+
+  const data = response.data;
+
+  if (!data.success) {
+    throw new Error('Mempool API failure');
```

Comment:
Maybe log `data` so we can see the response on the logs

Reply:
Done.